apps/dashboard/src/components/analytics/auto-bar-chart.tsx

```diff
@@ -81,19 +81,18 @@ export const AutoBarChart = <
   showXAxis,
   showYAxis,
   stacked,
-  ...boxProps
 }: AutoBarChartProps<TData, TIndexKey>) => {
-  const { colorMode } = useColorMode();
+  const { theme } = useTheme();
```

Comment:
`next-themes` === our new global theme source of truth, yes?

Reply:
Yes ( since we don't have chakra in app router pages )